include/coreinit/performancemonitor.h

```diff
@@ -0,0 +1,206 @@
+#pragma once
+#include <wut.h>
+
+/**
+ * \defgroup coreinit_performancemonitor Performance Monitor
+ * \ingroup coreinit
+ *
+ * @{
+ */
+
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+
+/// Used to tell OSSetPerformanceMonitor() which arguments are valid.
+typedef enum PerfMonArg {
+    PM_ARG_MMCR0 = 1u << 0u,
+    PM_ARG_MMCR1 = 1u << 1u,
+    PM_ARG_PMC1  = 1u << 2u,
+    PM_ARG_PMC2  = 1u << 3u,
+    PM_ARG_PMC3  = 1u << 4u,
+    PM_ARG_PMC4  = 1u << 5u,
```

Comment:
Can you prefix these and the other enum definitions with `OS_` as well?

Reply:
Done. And did the same for the flags.